.github/workflows/Documentation.yaml

```diff
@@ -78,6 +82,8 @@ jobs:
       IS_PROD: ${{ ! startsWith(needs.validate-documentation-request.outputs.oci-img-name, 'mock-') }}
     steps:
       - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.release-commit-sha}}
```

Comment:
```suggestion
          ref: ${{ inputs.release-commit-sha }}
```

Reply:
and, what happen on `workflow_dispatch`? Will this be empty or throw an error? and if empty, will the `checkout` action default to `main`?

Reply:
On `workflow_dispatch` the `inputs.release-commit-sha` will be empty. The `with: ref:` won't be effective. The checkout action will default to the branch, based on which the workflow is dispatched.